sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -150,7 +157,14 @@ internal open class KlaviyoApiRequest(
     /**
      * HTTP request headers
      */
-    override var headers: Map<String, String> = emptyMap()
+    override var headers: MutableMap<String, String> = mutableMapOf(
+        HEADER_CONTENT to TYPE_JSON,
+        HEADER_ACCEPT to TYPE_JSON,
+        HEADER_REVISION to V3_REVISION,
+        HEADER_USER_AGENT to DeviceProperties.userAgent,
+        HEADER_KLAVIYO_MOBILE to "1",
+        HEADER_KLAVIYO_ATTEMPT to "1/${Registry.config.networkMaxAttempts}"
```

Comment:
should this use attempts instead of hardcoded 1?

Reply:
Yeah lemme see about that. I also considered not even putting it in here as a default, since it is always added/updated at send time.

Reply:
Will check. Its certainly covered by other tests, but it'd be good to have a standalone test for that.

Reply:
also, when we persist requests to disk, have we also accounted for storing the attempts? or are we saying, if the app restarts, we'll reset the attempts to 0?

Reply:
Attempts is not persisted right now on either platform

Reply:
Cut a ticket for discussion about persisting attempt count to disk. I think it makes more sense to be persisted, its probably how we _intended_ that 50 retry limit to behave. Best we have a team chat about this though, since it hasn't been working that way up till now.